src/main/java/Duke.java

```diff
@@ -1,10 +1,133 @@
+import java.util.Scanner;
+
 public class Duke {
+    // Line width
+    private static final int LINE_WIDTH = 60;
+
+    // Command strings
+    private static final String COMMAND_TODO = "todo";
+    private static final String COMMAND_DEADLINE = "deadline";
+    private static final String COMMAND_EVENT = "event";
+    private static final String COMMAND_LIST = "list";
+    private static final String COMMAND_DONE = "done";
+    private static final String COMMAND_BYE = "bye";
+
+    // Print horizontal line for improving readability
+    private static void printHorizontalLine() {
+        System.out.println("_".repeat(LINE_WIDTH));
+    }
+
+    // Print greeting message
+    private static void printGreeting() {
```

Comment:
I like how you placed the strings to be printed in a function

Reply:
Thank you!